doc/man3/CMS_EnvelopedData_create.pod

```diff
@@ -47,6 +53,24 @@ CMS_EnvelopedData_create_ex() and CMS_AuthEnvelopedData_create_ex()
 but use default values of NULL for
 the library context I<libctx> and the property query I<propq>.
 
+CMS_sign_encrypt() creates a B<CMS_EnvelopedData> structure for recipients in
+I<enc_recip>.
+
+I<data> is signed using I<signcert> and I<signkey> to
+create B<CMS_SignedData> and then encrypted using I<enc_recip> to
+create B<CMS_EnvelopedData>. The library context I<libctx> and the property
```

Comment:
As mentioned earlier, in POD files after a fullstop (end of sentence) better start a new line.
```suggestion
I<data> is signed using I<signcert> and I<signkey> to create B<CMS_SignedData> 
and then encrypted using I<enc_recip> to create B<CMS_EnvelopedData>.
The library context I<libctx> and the property
```